### PR TITLE
Vis network

### PR DESCRIPTION
**Pull request by setaman**

### CODE REVIEW

**Comment by setaman** [comment]
@DaTebe addresses more or less #223?

File: web-client/src/api/keycloak.js [hunk 1]
```diff
@@ -35,8 +35,8 @@ export default {
   init: () =>
     kc
       .init({
-        onLoad: "check-sso",
-        silentCheckSsoRedirectUri: window.location.href,
+        onLoad: "login-required",
```
Comment (DaTebe) [inline web-client/src/api/keycloak.js]: This should no be here

File: web-client/src/components/DataNetwork/DataNetwork.vue [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    on(event, callback) {
+      this.network.on(event, callback);
+    },
+    off(event, callback) {
+      this.network.off(event, callback);
+    },
+    once(event, callback) {
+      this.network.once(event, callback);
+    },
+    enableEditMode() {
```
Comment (DaTebe) [inline web-client/src/components/DataNetwork/DataNetwork.vue]: Are these methods used?

File: web-client/src/components/DataNetwork/EntityDataNetwork.vue [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    };
+  },
+};
+</script>
+
+<style scoped lang="scss">
+div.datanetwork {
+  @include border-radius;
+  overflow: hidden;
+  height: calc(100vh - 315px);
```
Comment (DaTebe) [inline web-client/src/components/DataNetwork/EntityDataNetwork.vue:269]: vh - px? Where does 315 come from?

**Review by DaTebe — APPROVED** [review]

**Pull request merged**